docs/default_csi_plugin.md

```diff
@@ -19,6 +19,13 @@ boots. [StorageCapacity tracking](https://kubernetes.io/docs/concepts/storage/st
 Pods with an ODF-LVM PVC are not scheduled if the requested storage is greater than the volume group's remaining free
 storage.
 
+> **IMPORTANT!** StorageCapacity tracking poses a race condition in which the node's storage capacity is not up-to-date
```

Comment:
Is this problem unique to MicroShift, or does it apply to LVMO on SNO, too? If there is no unique MicroShift issue, we should refer the user to LVMO docs that talk about the problem.

Reply:
It's a known issue with TopoLVM when deployed in this topology (without the extended scheduler).  LVMO does deploy these extended scheduler, and so doesn't encounter this.  IDK what SNO's deployment mode is for topolvm.  If it's managed via the LVM Operator, then it'll also have the extended scheduler running.  We opted not to run the scheduler on MicroShift to save space and because it's a single node deployment.

Reply:
The main reason we've chosen TopoLVM over HPP is so it correctly isolates PVs from each other and guarantees allocated storage. If it doesn't do that we need to consider this a bug.